lib/child-writer/index.js

```diff
@@ -136,7 +134,7 @@ exports.write = (image, drive, options) => {
       // Don't be so strict. If a message doesn't look like
       // a robot message, then make the child writer log it
       // for debugging purposes.
-      return robot.parseMessage(robot.buildMessage(ROBOT_COMMAND_LOG, {
+      return robot.parseMessage(robot.buildMessage(robot.COMMANDS.LOG, {
```

Comment:
To stick with the conventional naming of enums, should this be singular, rather than plural, i.e. `robot.COMMAND.LOG`?